SECURITY.md

```diff
@@ -1,3 +1,11 @@
-# Security and Vulnerability Reporting
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please read the guidelines on reporting security issues [on the
+official website](https://www.python.org/dev/security/) for
+instructions on how to report a security-related problem to
+the Python Security Response Team responsibly.
+
+To reach the response team, email `security at python dot org`.
 
```

Comment:
Trim the double-trailing newlines please!

Reply:
Done